src/UBAFeeCalculator/UBAFeeUtility.ts

```diff
@@ -293,16 +293,44 @@ export function calculateUtilization(
   hubBalance: BigNumber,
   hubEquity: BigNumber,
   ethSpokeBalance: BigNumber,
-  spokeTargets: { target: BigNumber; spokeChainId: number }[]
+  spokeTargets: { target: BigNumber; spokeChainId: number }[],
+  hubPoolChainId = HUBPOOL_CHAIN_ID
 ) {
   const numerator = hubBalance
     .add(ethSpokeBalance)
-    .add(spokeTargets.reduce((a, b) => (b.spokeChainId !== HUBPOOL_CHAIN_ID ? a.add(b.target) : a), BigNumber.from(0)));
+    .add(spokeTargets.reduce((a, b) => (b.spokeChainId !== hubPoolChainId ? a.add(b.target) : a), BigNumber.from(0)));
   const denominator = hubEquity;
   const result = numerator.mul(parseEther("1.0")).div(denominator); // We need to multiply by 1e18 to get the correct precision for the result
   return BigNumber.from(10).pow(decimals).sub(result);
 }
 
+export function calculateUtilizationBoundaries(
+  action: {
+    actionType: "deposit" | "refund";
```

Comment:
should "deposit" and "refund" be added to some `enum` if we use them elsewhere?

Reply:
Good call - updated: https://github.com/across-protocol/sdk-v2/pull/236/commits/65cb9a0ebe0e99e47ff2d71bee03fa486ab54fc1